CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 ### Added 🚀
 
 -   A changelog dialog with the latest additions to CodeCharta appears on version update ([#1315](https://github.com/MaibornWolff/codecharta/issues/1315))
+-   An option has been added to the global configuration to enable coping screenshot to clipboard instead of saving it in a file ([#2326](https://github.com/MaibornWolff/codecharta/issues/2326))
```

Comment:
```suggestion
-   An option has been added to the global settings to enable copying screenshots to clipboard instead of saving them in a file ([#2326](https://github.com/MaibornWolff/codecharta/issues/2326))
```